src/core_plugins/metric_vis/public/metric_vis_params.html

```diff
@@ -151,12 +193,20 @@
               ng-options="mode for mode in collections.colorSchemas"
```

Comment:
Don't we want to have the color scheme names translated? Since it have values as `Green to red`, `Blues`, `Yellow to red`, etc.? cc @alexfrancoeur

Reply:
The color scheme names are used in 5 files more. To have the names translated we need to update all 6 files and make sure that the functionality is not broken. I think it's worth to create a separate PR for this purpose. Are you in agreement with it?